### PR TITLE
Update view-quote designs to better represent the metamask fee

### PR DESCRIPTION
**Pull request by danjm**
This PR updates view-quote to match latest designs

-

-

-

### CODE REVIEW

File: ui/app/pages/swaps/view-quote/view-quote.js [hunk 1]
```diff
@@ -494,6 +496,15 @@ export default function ViewQuote () {
             <i className="fa fa-arrow-right" />
           </div>
         </div>
+        <div className="view-quote__metamask-rate">
+          <div className="view-quote__metamask-rate-text">{ t('swapQuoteIncludesRate', [metaMaskFee]) }</div>
+          <div className="view-quote__metamask-rate-info-icon">
```
Comment (brad-decker) [inline ui/app/pages/swaps/view-quote/view-quote.js:501]: req: collapse this extra div into the div that will be rendered around the tooltip by using `wrapperClassName` from `InfoTooltip`

File: ui/app/pages/swaps/view-quote/view-quote.js [hunk 2]
```diff
@@ -494,6 +496,15 @@ export default function ViewQuote () {
             <i className="fa fa-arrow-right" />
           </div>
         </div>
+        <div className="view-quote__metamask-rate">
+          <div className="view-quote__metamask-rate-text">{ t('swapQuoteIncludesRate', [metaMaskFee]) }</div>
```
Comment (brad-decker) [inline ui/app/pages/swaps/view-quote/view-quote.js:500]: question: Should this be a paragraph instead of a div?

**Review by brad-decker — CHANGES_REQUESTED** [review]
LGTM, needs a few tweaks imo but should be straightforward.

**Comment by danjm** [comment]
@brad-decker your comments have been addressed in the latest commit

**Review by brad-decker — DISMISSED** [review]

**Comment by jakehaugen** [comment]
Looks great. Two small copy changes:

1. "best swap fee" to "MetaMask fee"
2. Tooltip copy update: "We find the best price from the top liquidity sources, every time. A fee of 0.875% is automatically factored into each quote, which supports ongoing development to make MetaMask even better."

**Comment by danjm** [comment]
The copy updates requested by @jakehaugen have been included in the latest commit https://github.com/MetaMask/metamask-extension/pull/9633/commits/ed06da8adb13af14ac1d6563fe793f17c91b1be5

**Review by brad-decker — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

File: app/_locales/en/messages.json [hunk 3]
```diff
@@ -1729,7 +1729,7 @@
     "message": "MetaMask fee"
   },
   "swapMetaMaskFeeDescription": {
-    "message": "A service fee of $1% is automatically factored into each quote, which supports ongoing development to make MetaMask even better.",
+    "message": "We find the best price from the top liquidity sources, every time. A fee of $1% is automatically factored into each quote, which supports ongoing development to make MetaMask even better.",
```
Comment (Gudahtt) [inline app/_locales/en/messages.json:1732]: Is the percent sign being duplicated here? I noticed that it's in both of these messages _and_ in the `metaMaskFee` variable passed in.

**Pull request merged**